packet.go

```diff
@@ -109,6 +109,11 @@ func parsePacket(i *astikit.BytesIterator) (p *Packet, err error) {
 		}
 	}
 
+	// Use optional packet filter
+	if flt != nil && flt(p) {
+		return nil, nil
```

Comment:
Could you add a `var errSkippedPacket = errors.New("astits: skipped packet")` at the top of this file and return `nil, errSkippedPacket` when packet is skipped?

I'd rather indicate to the caller that packet has been skipped.

Reply:
Should we pass this error to NextData() caller or can we  just `continue` loop if such error occur?
In my opinion NextData() caller don't care much about transport part of protocol as long as data collected.

Reply:
Or this error only for internal purpose between packetBuffer.next() and parsePacket()?

Reply:
You can keep it as it is, in my mind this error was only intended to the caller of `parsePacket()`